sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.2.0-beta.4 (Unreleased)
+## 1.2.0 (Unreleased)
```

Comment:
Please specify the release date here by replacing the `Unreleased`.

Reply:
fixed